schemas/layer-3.cue

```diff
@@ -0,0 +1,67 @@
+// layer-3-policy.cue
+
+// TODO
+// Create schema for a policy document
+// Policy docs should reference common and unique risk definitions, so we need schema for that as well
+
+#Policy: {
+    // Metadata useful for evaluation and automation
+    metadata?: {
+        version: string
+        author: string
+        last_modified: string
+        sci_version: string
+        remarks: string
+    }
+
+    // Unique identifier for this policy
+    id: string
+
+    // Human-readable title of the policy
+    title: string
+
+    // Short description of this policy’s intent or purpose
+    description: string
+
+    // Optional reference to a parent policy this one inherits from or refines
+    parent_policy_id?: string
+
+    // Policy classification level (e.g., mandatory, recommended)
+    classification: oneOf("mandatory", "recommended")
```

Comment:
I don't see `oneOf` in https://cuelang.org/docs/reference/spec/#builtin-functions

Should this be
```suggestion
    classification: "mandatory" | "recommended"
```